pkgs/by-name/gu/gungnir/package.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "gungnir";
+  version = "1.0.8";
+
+  src = fetchFromGitHub {
+    owner = "g0ldencybersec";
+    repo = "gungnir";
+    rev = "v${version}";
+    hash = "sha256-k6fxAvUBAAcTHzdeGhekYhPpnS05jHq/7EqxafQfMio=";
+  };
+
+  vendorHash = "sha256-r2aU59L0fnSdc/lpR04K/GQ1eZ7ihV+tKlyuS6sPX2o=";
+
+  ldflags = [ "-s" "-w" ];
+
+  meta = with lib; {
+    description = "command-line tool that continuously monitors certificate transparency (CT) logs for newly issued SSL/TLS certificates.";
```

according to <https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes> you're missing capitalization and no period at the end
```suggestion
    description = "A command-line tool that continuously monitors certificate transparency (CT) logs for newly issued SSL/TLS certificates";
```

---

Updated, thanks! Seems like I overlooked the part with the conventions for the meta attributes ^^